src/mirco_setparameters.cpp

```diff
@@ -51,8 +51,18 @@ void MIRCO::SetParameters(double& E1, double& E2, double& LateralLength, double&
   // Composite Poisson's ratio
   CompositePoissonsRatio = CompositeYoungs / (2 * CompositeShear) - 1;
 
-  // Correction factor vector
-  std::vector<double> alpha_con{0.778958541513360, 0.805513388666376, 0.826126871395416,
+  // Correction factor vectors
+  // These are the correction factors to calculate the elastic compliance of the micro-scale contact
+  // constitutive law for various resolutions.
+  // The following pressure based constants are calculated by solving a flat indentor problem using
+  // the pressure based Green function described in Pohrt and Li (2014).
+  // http://dx.doi.org/10.1134/s1029959914040109
+  std::vector<double> alpha_con_pressure{0.961389237917602, 0.924715342432435, 0.899837531880697,
```

Comment:
What does `alpha_con` stand for?

Also, you can make these vectors `const`.

Reply:
This variable name has not been changed since the beginning. The Greek letter `alpha` is used in Section 3.3 of https://doi.org/10.1007/s00466-019-01791-3 to denote this variable. These are called "shape factors". Once we have the shape factors, the Elastic compliance correction can be calculated as done in line 89 of the `mirco_set_parameters.cpp`. I will rename this variable for a better readability of the code.